Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs

```diff
@@ -117,10 +122,38 @@ private Vector2 GetScreenResoultion()
         private void Awake()
         {
             _logger = PatcherLogManager.DefaultLogger;
+            _canvasScaler = GetComponent<CanvasScaler>();
+
+            float scaler;
+            float currentDpi = Screen.dpi;
+            if (currentDpi == 0)
+            {
+                scaler = 1;
+                _logger.LogWarning("Unable to determine the current DPI.");
+            }
+            else if (currentDpi > 200 && currentDpi < 400)
+            {
+                scaler = 2;
+            }
+            else if (currentDpi >= 400)
+            {
+                scaler = 4;
+            }
+            else
+            {
+                scaler = 1;
+            }
```

Comment:
I'd recommend different flow of condition checking

```
if (currentDpi >= 400)
{
    ...
}
else if (currentDpi >= 200)
{
    ...
}
else
{
    if (currentDpi == 0)
    {
        ... (log warning)
    }

    ...
}
```

Reply:
done